pkg/userdata/centos/provider.go

```diff
@@ -188,6 +186,7 @@ write_files:
 
     yum install -y yum-utils
     yum-config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+    yum-config-manager --save --setopt=docker-ce-stable.module_hotfixes=true
```

Comment:
Same here, I think a comment would be useful to avoid forgetting about why this is needed.

Reply:
👍